pkg/controllers/manager.go

```diff
@@ -54,16 +59,22 @@ func NewManagerOrDie(config *rest.Config, options controllerruntime.Options) Man
 
 // RegisterControllers registers a set of controllers to the controller manager
 func (m *GenericControllerManager) RegisterControllers(controllers ...Controller) Manager {
-	for _, controller := range controllers {
-		controlledObject := controller.For()
-		var builder = controllerruntime.NewControllerManagedBy(m).For(controlledObject)
-		if namedController, ok := controller.(NamedController); ok {
+	for _, c := range controllers {
+		controlledObject := c.For()
+		var builder = controllerruntime.NewControllerManagedBy(m).For(controlledObject).WithOptions(controller.Options{
```

Comment:
IIUC controllerruntime's default rate limiter is this one: https://github.com/kubernetes/client-go/blob/0c029ff945dd24c074f455220b9a4ed0506f5f4d/util/workqueue/default_rate_limiters.go#L39.

I think we never want 10QPS on the bucket limiter. We probably want a max of 1 QPS if we configure it at all. Agree that we want to slow down the max backoff to 10 seconds rather than 1 second.

Reply:
I'll put it back to the enumerated default. 👍